### PR TITLE
Add meshsync status logo in header

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
Signed-off-by: Prakhar Agarwal <prakhar@everything.net.in>

**Description**

This PR fixes #

**Notes for Reviewers**
This PR adds meshsync logo which changes according to graphql subscription. It completes the UI part of #4712

**Screenshot**
See the meshsync logo in header (top-right)

Screenshot from 2021-12-18 13-25-39

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Prakhar-Agarwal-byte** [comment]
These tests shouldn't fail 🤔

**Comment by warunicorn19** [comment]
@Prakhar-Agarwal-byte these are not from your changes, it's a cypress issue that is being worked on, I'll re-run the jobs just to be double sure.

**Review by leecalcote — CHANGES_REQUESTED** [review]
@Prakhar-Agarwal-byte will you add a tooltip on hover that shows the status and/or version number of MeshSync? Will you add a hyperlink to the MeshSync logo, such that when it is clicked, the user is navigated to /settings#environment?

File: ui/components/Header.js [hunk 1]
```diff
@@ -176,13 +176,9 @@ class Header extends React.Component {
                     <MesheryNotification />
                   </div>
 
-                  {/* <Tooltip title="MeshSync Status">
-                    <div style={{ padding : "1rem", height : "2rem", width : "2rem", borderRadius : "50%", backgroundColor : this.state.meshSyncStatus.status === "ENABLED" ? "green" : "red" }} />
-                  </Tooltip>
-
-                  <Tooltip title="Broker Status">
-                    <div style={{ padding : "1rem", height : "2rem", width : "2rem", borderRadius : "50%", backgroundColor : this.state.brokerStatus ? "green" : "red" }} />
-                  </Tooltip> */}
```
Comment (bariqhibat) [inline ui/components/Header.js:185]: Are we not showing the broker status logo as well? It's different from MeshSync

**Review by bariqhibat — CHANGES_REQUESTED** [review]
UI-wise, it's good! But just a comment on the Broker Logo, are we missing that part on Figma or you'll introduce it in another PR? Also, do address the comment by @leecalcote

**Comment by warunicorn19** [comment]
I think we're missing designs for the broker logo, I can't seem to find it on Figma. Wouldn't putting it next to the meshsync logo make the header a bit too crowded?

**Comment by leecalcote** [comment]
Right. NATS status will be incorporated into the MeshSync logo, so we will only have the one logo. The two logos and statues will be separate on the /settings page, though. 

There are additional enhancements to make here, but this PR (when the hyperlink is added) is a solid first step (given how it connects to the graphql-based status that @bariqhibat created).

File: ui/components/Header.js (hunk 1 above)
Comment (leecalcote) [reply ui/components/Header.js:185]: Right - a different PR. 👍

**Pull request merged**